server/routes/follower.js

```diff
@@ -0,0 +1,91 @@
+const { Op } = require("sequelize");
+const express = require("express");
+const router = express.Router();
+const authMiddleware = require("../middlewares/authMiddleware");
+const {
+  followerIdCheck,
+  validate,
+  giveServerError,
+} = require("../middlewares/validate");
+const { UserModel, UserFollowerModel } = require("../models/models");
+const db = require("../models");
+
+//Gets all the users the user is currently following, and from that gets all the users that aren't part of that group.
+const getStrangers = async (req, res) => {
```

Comment:
lol let's call all references to strangers `suggestions` instead. The idea eventually will be that this is a list of suggested users to follow